docs/questions.rst

```diff
@@ -158,6 +158,27 @@ Metadata Specification and are supposed to be synced temporarily
 to keep backward compatibility.
 
 
+.. _restraint-compatibility:
+
+Restraint Compatibility
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For backward-compatibility ``tmt`` provides selected commands
+of the `restraint`__ framework so that existing tests can be more
+easily migrated. Currently the following scripts are supported:
+
+* ``rhts-abort`` and ``rhts-abort`` — :ref:`/stories/features/abort`
```

Comment:
Maybe I misunderstood, but why is each command listed twice? Shouldn't one of them have the `rstrnt` prefix instead of `rhts`?

Reply:
Ah! Good catch! Apparently a bunch of us have a blind spot there ;-) Should be fixed in bf58e1f1.